plugins/destination/mssql/queries/templates/drop_table.sql.tpl

```diff
@@ -0,0 +1 @@
+DROP TABLE {{.Table}};
```

Comment:
We might need to re-consider some of these templates. I think adding a template each time you want to write a query is a bit much. Not sure what is the added value of having those instead of doing it in Go code directly

Reply:
I'm OK with removing the templates altogether, as was done in CH dst (initially implemented with templates but then moved from them)

Reply:
Yeah, no need for it now. Maybe in a future refactor